src/components/order-service/order-service-info.html

```diff
@@ -0,0 +1,32 @@
+<div class="order-service-details">
+  <div class="order-service-details-top">
+    <div class="service-icon">
+      <span ng-if="!$ctrl.imageUrl" class="icon {{$ctrl.iconClass}}"></span>
+      <span ng-if="$ctrl.imageUrl" class="image"><img ng-src="{{$ctrl.imageUrl}}" alt=""></span>
+    </div>
+    <div class="service-title-area">
+      <div class="service-title">
+        {{$ctrl.serviceName}}
+      </div>
+      <div ng-if="$ctrl.serviceClass.tags" class="order-service-tags">
+        <span ng-repeat="tag in $ctrl.serviceClass.tags" class="tag">
+          {{tag}}
+        </span>
+      </div>
+      <div ng-if="$ctrl.serviceClass.resource.externalMetadata.documentationUrl" class="order-service-documentation-url">
+        <a ng-href="{{$ctrl.serviceClass.resource.externalMetadata.documentationUrl}}" target="_blank" class="learn-more-link">Learn More <i class="fa fa-external-link" aria-hidden="true"></i></a>
+      </div>
```

Comment:
We should probably add `externalMetadata.supportUrl` too

Reply:
![screen shot 2017-09-21 at 12 08 47 pm](https://user-images.githubusercontent.com/895728/30706264-d6561020-9ec5-11e7-848e-d46e275915e8.PNG)